library/src/test/java/com/karumi/headerrecyclerview/HeaderRecyclerViewAdapterBaseTest.java

```diff
@@ -261,4 +353,6 @@ public void shouldReturnFirstItemAsItemAtPositionOneIfHeaderWasConfiguredAndTher
   protected abstract F givenAFooter();
 
   protected abstract List<T> givenAListWithFiveItems();
+
+  protected abstract VH givenAMockViewHolder();
```

Comment:
`givenAViewHolder()` return a not mockable object, I am not able use Powermock to mock `ViewHolder.getAdapterPosition()`